src/app/authentication/sign-in/sign-in.component.ts

```diff
@@ -10,10 +11,22 @@ export class SignInComponent implements OnInit {
 
   user: User = {} as User;
 
-  constructor() {
+  constructor(public authService: AuthenticationService) {
```

Comment:
Should be private if not used in the template

Reply:
ok.